spec/passaporte_web/service_account_member_spec.rb

```diff
@@ -5,8 +5,8 @@
   let(:service_account) { PassaporteWeb::ServiceAccount.find('859d3542-84d6-4909-b1bd-4f43c1312065') }
   let(:identity) { PassaporteWeb::Identity.find('5e32f927-c4ab-404e-a91c-b2abc05afb56') }
 
-  let(:mock_service_account) { mock('ServiceAccount', uuid: 'service-account-uuid') }
-  let(:mock_identity) { mock('Identity', uuid: 'identity-uuid') }
+  let(:mock_service_account) { double('ServiceAccount', uuid: 'service-account-uuid') }
+  let(:mock_identity) { double('Identity', uuid: 'identity-uuid') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.